core/src/main/scala/akka/kafka/scaladsl/PartitionAssignmentHandler.scala

```diff
@@ -20,15 +20,15 @@ import org.apache.kafka.common.TopicPartition
  * the callbacks in this class are called.
  *
  * This complements the methods of Kafka's [[org.apache.kafka.clients.consumer.ConsumerRebalanceListener ConsumerRebalanceListener]] with
- * an `onStop` callback.
+ * an `onStop` callback which is called before `Consumer.close`.
  */
 @ApiMayChange
 trait PartitionAssignmentHandler {
 
   /**
    * See [[org.apache.kafka.clients.consumer.ConsumerRebalanceListener#onPartitionsRevoked]]
    *
-   * @param revokedTps The list of partitions that were assigned to the consumer on the last rebalance
+   * @param revokedTps The list of partitions that were revoked from the consumer
    * @param consumer A restricted version of the internally used [[org.apache.kafka.clients.consumer.Consumer Consumer]]
```

Comment:
It's not really a `KafkaConsumer` though in the `is a` sense.  Should we link to `RestrictedConsumer` instead, or in addition, to this?